installing/installing_bare_metal_single_node/install-sno-preparing-to-install-sno.adoc

```diff
@@ -0,0 +1,20 @@
+[id="preparing-to-install-sno"]
+= Preparing to install single node OpenShift (SNO)
```

Comment:
Remove "OpenShift" from the title.  Maybe "Preparing to install on a single node".

Reply:
"Preparing to install on a single node"--won't that match the _topic-map.yml entry?

Reply:
If you do not need this file, `git rm` it.  If this file is used in the doc, please make the changes to refer to "single node" or "single node cluster" like you did in `installing/installing_sno/install-sno-preparing-to-install-sno.adoc`.

Reply:
Copy the file name, control + F, and search. You will see that I have made this change and checked it in. For whatever reason, github is showing the first commit, and later showing the change as a new file instead of a diff. Perhaps this is due to squashing commits. However, the change is already in. How would you like to resolve this? I can cancel the PR and create a new one if that helps. @vikram-redhat